packages/webapp/src/containers/Animals/Inventory/index.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import { useDispatch } from 'react-redux';
+import {
+  useGetAnimalsQuery,
+  useGetAnimalBatchesQuery,
+  useGetAnimalGroupsQuery,
+  api,
+} from '../../../store/api/apiSlice';
+import Layout from '../../../components/Layout';
+import { Title } from '../../../components/Typography';
+
+function AnimalInventory() {
+  const dispatch = useDispatch();
+
+  // For demonstration only! (remove to observe caching); force refresh via:
+  dispatch(api.util.invalidateTags(['Animals', 'AnimalBatches', 'AnimalGroups']));
+
+  const { data: animals } = useGetAnimalsQuery();
+  const { data: animalBatches } = useGetAnimalBatchesQuery();
+  const { data: animalGroups } = useGetAnimalGroupsQuery();
+
```

Comment:
This is the point at which type-checking will be enforced on the returned data based on the interfaces that were provided in `apiSlice.ts`-- it's fun to see by trying to access existent + non-existent properties on each of those data objects.

Although these can be successfully batched with the wrapper function `useQueries` linked above:

```ts
const { data, isLoading } = useQueries([
  { hook: useGetAnimalsQuery, label: 'animals' },
  { hook: useGetAnimalBatchesQuery, label: 'animalBatches' },
  { hook: useGetAnimalGroupsQuery, label: 'animalGroups' },
]);

const { animals, animalBatches, animalGroups } = data;
```

I never figured out how to modify the wrapper to still get that type checking.